### ISSUE TITLE
Slight difference in __init__.py files

### ISSUE THREAD

**Issue by DManowitz** [Kitware/trame#420]
**Describe the bug**
I noticed that each of the trame packages provides its own `__init__.py` files for the main trame directory and several of the subdirectories where they add files.  For the most part, these files are identical.  However, I have noticed a couple of variations, although I'm not sure if they actually result in different behaviors.

In particular, most of the `__init__.py` files contain the following line:
```
__path__ = __import__("pkgutil").extend_path(__path__, __name__)
```

However, in `trame/modules` from `trame-client`, the `__init__.py` file has the following line:
```
__import__("pkg_resources").declare_namespace(__name__)
```

I've also noticed that the above lines are present in reverse between `trame/ui` from `trame` vs `trame/ui` from `trame-router`.

Finally, as a very minor difference, the `__init__.py` file in the main `trame` directory from `trame` includes a `__license__` string, but this is missing from the `__init__.py` files put in the main `trame` directory from the other trame packages.

**Comment by jourdain** [comment]
@psavery

**Comment by psavery** [comment]
The [official python documentation](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages) indicates that the `pkgutil` version "is the recommended approach for the highest level of compatibility." The documentation also indicates that the `pkgutil` and `pkg_resources` versions are not compatible with one another, so we should definitely be consistent. We should change to the `pkgutil` version in all cases.

Another benefit of using the `pkgutil` version instead of the `pkg_resources` version is that `pkg_resources` comes from `setuptools`, which is longer included (by default) in Python3 as of Python3.12. So if we use the `pkgutil` version, we can avoid adding a dependency on `setuptools`.

The namespace line (`__path__ = __import__("pkgutil").extend_path(__path__, __name__)`) should be the only line in those `__init__.py` files. Any other lines have a good chance of not being accessible after installation. So we should remove any extra lines as well (such as `__license__`).

**Comment by jourdain** [comment]
Should now be fixed in `trame>=3.5.3` and `trame-client>=2.16.3`

**Comment by jourdain** [comment]
`trame-vuetify>=2.4.3` also fix some inconsitency
